### PR TITLE
feat: sync HTML entities

### PR DESCRIPTION
**Pull request by nschonni**
From https://html.spec.whatwg.org/multipage/named-characters.html#named-character-references
No includes the different casings since dictionaries seem to be going case sensitive

### CODE REVIEW

File: dictionaries/html-symbol-entities/entities.txt [hunk 1]
```diff
@@ -1,25 +1,51 @@
+&Aacute;
+&Aacute
```
Comment (nschonni) [inline dictionaries/html-symbol-entities/entities.txt]: Getting dups from the non-trailing semicolons

File: dictionaries/html-symbol-entities/entities.txt [hunk 2]
```diff
@@ -1,25 +1,51 @@
+&Aacute;
```
Comment (nschonni) [inline dictionaries/html-symbol-entities/entities.txt:1]: Should the leading/trailing punctuation be stripped off?

Comment (Jason3S) [reply dictionaries/html-symbol-entities/entities.txt:1]: Yes, they should be removed.

The tool to build the dictionary can strip them off, but I think it is better to just remove them.

**Comment by Jason3S** [comment]
Thank you.

**Comment by Jason3S** [comment]
This PR is part of #1034

**Pull request closed without merging**